### ISSUE TITLE
Custom Links with `#` are not processed correctly

### ISSUE THREAD

**Issue by carlosaya** [drewhammond/chefbrowser#422]
Using the following custom link results in the rendered link on the page being malformed:

```
[custom_links.nodes.0]
title = "Open in Launch"
href = "https://hostname/automation/#/service/catalog/consume/deployment?search=%22{{name}}%22"
new_tab = true
```

The resulting link rendered on the page points to `http://localhost:8080/ui/nodes/%22https://hostname/automation/`.  URL's without the `#` character are processed correctly and function as expected.

**Comment by drewhammond** [comment]
This sounds like what I fixed in https://github.com/drewhammond/chefbrowser/pull/400 but never cut a release. Will have this out later today in 0.1.7. Thanks!

**Comment by carlosaya** [comment]
No, thank you!

**Comment by drewhammond** [comment]
Yeah, that's basically what it was doing under the hood. The fix in #400 isn't perfect because ideally I'd like to be able to support both inline comments and quoted strings containing `#`, but it'll do for now.

Released in [0.1.7](https://github.com/drewhammond/chefbrowser/releases/tag/0.1.7)!
